source/includes/dev/_deletion.md

```diff
@@ -0,0 +1,353 @@
+
+> Create a deletion for the current user
+
+```shell
+curl -X POST "https://api.resourcewatch.org/v1/deletion" \
+-H "Authorization: Bearer <your-token>"
+```
+
+> Response:
+
+```json
+{
+  "data": {
+    "type": "deletions",
+    "id": "62bd8d49cae089972ce81039",
+    "attributes": {
+      "userId": "62bd8d49cae089972ce81038",
+      "requestorUserId": "62bd8d49cae089972ce81038",
+      "status": "pending",
+      "datasetsDeleted": false,
+      "layersDeleted": false,
+      "widgetsDeleted": false,
+      "userAccountDeleted": false,
+      "userDataDeleted": false,
+      "graphDataDeleted": false,
+      "collectionsDeleted": false,
+      "favouritesDeleted": false,
+      "vocabulariesDeleted": false,
+      "areasDeleted": false,
+      "storiesDeleted": false,
+      "subscriptionsDeleted": false,
+      "dashboardsDeleted": false,
+      "profilesDeleted": false,
+      "topicsDeleted": false,
+      "createdAt": "2022-06-30T11:47:21.305Z",
+      "updatedAt": "2022-06-30T11:47:21.305Z"
+    }
+  }
+}
+```
+
+> Create a deletion for a specific user with custom data
+
+```shell
+curl -X POST "https://api.resourcewatch.org/v1/deletion" \
+-H "Content-Type: application/json" \
+-H "Authorization: Bearer <your-token>" \
+-d  \
+'{
+    "userId": "bd8a36df-2e52-4b2d-b7be-a48bdcd7c769",
+    "userAccountDeleted": true
+}'
+```
+
+Use this endpoint to create a deletion record for the user identified in the token. Note that this does not delete any
+user account or data - you should use the [delete user endpoint](/doc-api/reference.html#deleting-a-user) to achieve
+that.
+
+You can optionally specify the `userId` or any of the boolean type fields values in the body of your request.
+The `requestorUserId`
+is automatically set based on the user token passed as a request header. The `status` field is set to `pending` by
+default.
+
+### Errors for creating a deletion
+
+| Error code | Error message                         | Description                                               |
+|------------|---------------------------------------|-----------------------------------------------------------|
+| 400        | Deletion already exists for this user | There is already a deletion record for this userId.       |
+| 401        | Unauthorized                          | You need to be logged in to be able to create a deletion. |
+| 403        | Not authorized                        | You need to have the `ADMIN` role                         |
+
+
+## Updating a deletion
+
+> Updating a deletion
+
+```shell
+curl -X PATCH "https://api.resourcewatch.org/v1/deletion/<deletion-id>" \
+-H "Authorization: Bearer <your-token>" \
+-H "Content-Type: application/json" -d \
+'{
+  "status":"done"
+}'
+```
+
+> Response:
+
+```json
+{
+  "data": {
+    "type": "deletions",
+    "id": "62bd8d49cae089972ce81039",
+    "attributes": {
+      "userId": "62bd8d49cae089972ce81038",
+      "requestorUserId": "62bd8d49cae089972ce81038",
+      "status": "done",
```

Comment:
I know this is just an example but shouldn't the delete status of all the individual resources have to be set to `true` to be able to set overall deletion status to `done`.